src/components/course/data/tests/utils.test.jsx

```diff
@@ -569,60 +577,63 @@ describe('getSubscriptionDisabledEnrollmentReasonType', () => {
 
   it.each([
     {
-      daysUntilExpirationIncludingRenewals: -17,
+      netDaysUntilExpiration: -17,
       hasEnterpriseAdminUsers: true,
       expectedReasonType: DISABLED_ENROLL_REASON_TYPES.SUBSCRIPTION_EXPIRED,
       catalogsWithCourse: [mockCatalogUuid],
     },
     {
-      daysUntilExpirationIncludingRenewals: -17,
+      netDaysUntilExpiration: -17,
       hasEnterpriseAdminUsers: false,
       expectedReasonType: DISABLED_ENROLL_REASON_TYPES.SUBSCRIPTION_EXPIRED_NO_ADMINS,
       catalogsWithCourse: [mockCatalogUuid],
     },
     {
-      daysUntilExpirationIncludingRenewals: -17,
+      netDaysUntilExpiration: -17,
       hasEnterpriseAdminUsers: true,
-      expectedReasonType: undefined,
+      expectedReasonType: DISABLED_ENROLL_REASON_TYPES.SUBSCRIPTION_EXPIRED,
       catalogsWithCourse: ['fake-catalog-uuid'],
     },
     {
-      daysUntilExpirationIncludingRenewals: 0,
+      netDaysUntilExpiration: 1,
       hasEnterpriseAdminUsers: true,
       expectedReasonType: undefined,
       catalogsWithCourse: [mockCatalogUuid],
     },
     {
-      daysUntilExpirationIncludingRenewals: 10,
+      netDaysUntilExpiration: 10,
       hasEnterpriseAdminUsers: true,
       expectedReasonType: undefined,
       catalogsWithCourse: [mockCatalogUuid],
     },
   ])('handles expired subscription: %s', ({
-    daysUntilExpirationIncludingRenewals,
+    netDaysUntilExpiration,
     hasEnterpriseAdminUsers,
     expectedReasonType,
     catalogsWithCourse,
   }) => {
-    const customerAgreementConfig = {
+    const customerAgreement = {
       subscriptions: [
         {
           enterpriseCatalogUuid: mockCatalogUuid,
-          daysUntilExpirationIncludingRenewals,
         },
       ],
+      netDaysUntilExpiration,
+      availableSubscriptionCatalogs: [],
     };
 
     const reasonType = getSubscriptionDisabledEnrollmentReasonType({
-      customerAgreementConfig,
+      customerAgreement,
       catalogsWithCourse,
       subscriptionLicense: undefined,
       hasEnterpriseAdminUsers,
     });
     expect(reasonType).toEqual(expectedReasonType);
   });
 
-  it.each([
+  // Test skipped due to the learner-licenses API no longer returning information related to exhausted seats
+  // TODO: once the learner-licenses api supports exhausted seats, reimplement and verify tests
```

Comment:
nit: `learner-licenses` itself never did include this info, hence the issue. Perhaps update to say "does not include information related to exhausted seats previously returned by a separate API that is no longer called.".